renovate.json5

```diff
@@ -66,5 +54,50 @@
       "matchPackageNames": ["node-fetch", "@types/node-fetch"],
       "allowedVersions": "2.x"
     },
+    /*
+      Docs-related Renovate rules
+      These ensure that the `docs/` folder, which is its own application with
+      its own dependencies which lives inside this repository, plays by its
+      own Renovate rules.  Those rules are defined within the external (npm)
+      package called `renovate-config-apollo-docs` (defined here only by
+      the `apollo-docs` suffix).
+    */
+    {
+      "matchPaths": [
+        "docs/package.json"
+      ],
+      "extends": [
+        "apollo-docs"
+      ],
+      // We need to tell Renovate to check the branches for each major version
+      // of Apollo Server.  "Past" major versions should be added here!
+      "baseBranches": [
+        "main",
+        "version-2"
+      ],
+    },
+    // The current Apollo Gatsby theme does not support a version of Gatsby
+    // that supports React 17.
+    {
+      "matchPaths": [
+        "docs/package.json"
+      ],
+      "matchPackageNames": ["react", "react-dom"],
+      "allowedVersions": "16.x",
+    },
+    // The current Apollo Gatsby theme does not support Gatsby v4
```

Comment:
Heh, I didn't even know there was a v4 :) Assuming we'll be catching up soon?

Reply:
We're most likely going to leave the Gatsby docs theme at v3 and sunset it after what we're currently working on goes out 😊